java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/internal/Expectations.java

```diff
@@ -365,11 +509,36 @@ void collectExpectedIssues(String comment, int line) {
     ParsedComment parseIssue(String comment, int line) {
       Matcher shiftMatcher = shift.matcher(comment);
       Matcher flowMatcher = FLOW.matcher(comment);
-      return createIssue(line,
+      ParsedComment parsedComment = createIssue(line,
         shiftMatcher.find() ? shiftMatcher.group(1) : null,
         comment,
         parseMessage(comment, flowMatcher.find() ? flowMatcher.start() : comment.length()),
         comment);
+
+      Issue attr = parsedComment.issue;
+      List<String> quickfixes = (List<String>) attr.get(QUICK_FIXES);
```

Comment:
this can be written `List<String> quickfixes = QUICK_FIXES.get(attr);`, which avoid the cast.